environment-gpu.yml

```diff
@@ -0,0 +1,23 @@
+dependencies:
+- python>=3.7,<3.8
+- colorama>=0.4.3,<0.5.0
+- numpy>=1.16,<2.0
+- opt_einsum>=3.1.0,<4.0.0
+- pytorch::pytorch==1.5.0
+- pytorch::torchvision==0.6.0
+- docopt>=0.6.2,<0.7.0
+- flake8>=3.7.7,<3.8.0
+- isort>=4.3.21,<4.4.0
+- pytest>=5.4.1,<5.5.0
+- pytest-cov>=2.8.1,<2.9.0
+- sphinx>=2.4.4,<2.5.0
+- pillow<7
+- pip
+- cudatoolkit=10.2.*
+- cudnn=7.6.*
+- pip:
+  - asttokens>=2.0.3,<3.0.0
+  - prettyprinter>=0.18.0,<0.19.0
+  - black>=19.10b0,<19.11.0
+  - pydocstyle==5.0.1
+  - gym>=0.17.1,<0.18.0
```

Comment:
I don't think we need the gpu stuff for the frontend.

Reply:
Hi ! I keep everything related to pytorch-gpu because pytorch backend is still in myia main package (ie. , even if `myia_backend_pytorch` is not installed, `myia` itself would still need GPU packages so that pytorch frontend can work).

Should I better change something here ?

Reply:
But the backend has its own requirements list (which should include the gpu stuff).  I don't see why you need it here.